### PR TITLE
Add truncate method to proto.TestRunner

### PR DESCRIPTION
**Pull request by ash-kamrip**
Resolves <#293

Due to some reasons I couldn't test the code , kindly let me know if any changes are required.

### CODE REVIEW

**Comment by ash-kamrip** [comment]
@prestonvasquez could you give it a try

**Comment by ash-kamrip** [comment]
@prestonvasquez  is it looking good now ?!

**Comment by ash-kamrip** [comment]
@prestonvasquez Hi Preston, could you let me know if any further changes are required ? I want to close this off before hacktoberfest ends?!

**Review by prestonvasquez — CHANGES_REQUESTED** [review]
@rangzen Need to fix the linting failures

File: internal/proto/test_runner.go [hunk 1]
```diff
@@ -171,6 +173,14 @@ func (runner *TestRunner) AddPingCases(cases ...TestCase) {
 	runner.pingCases = append(runner.pingCases, cases...)
 }
 
+// Adding cases for testing out the Truncate() storage method on multiple tables
+func (runner TestRunner) AddTruncateCases(cases ...TestCase) {
```
[the remark this answers is not in the document]
Comment (ash-kamrip) [reply internal/proto/test_runner.go]: Oopsies , wasn't able to run tests , i always got no tests while trying to run the tests , I missed this somehow. Thank you very much for helping out

File: internal/proto/test_runner.go [hunk 2]
```diff
@@ -171,6 +173,14 @@ func (runner *TestRunner) AddPingCases(cases ...TestCase) {
 	runner.pingCases = append(runner.pingCases, cases...)
 }
 
+// Adding cases for testing out the Truncate() storage method on multiple tables
```
[the remark this answers is not in the document]
Comment (ash-kamrip) [reply internal/proto/test_runner.go]: @prestonvasquez Thanks for this as well, not versed with the github actions and checking logs, I've learned them now thanks to your gidari and the time you've taken out to review this 😄

**Comment by rangzen** [comment]
@prestonvasquez Hello, sorry I wasn't available, do you still need something on this PR?

**Pull request merged**